src/smt/set_defaults.cpp

```diff
@@ -1401,24 +1401,6 @@ void setDefaults(SmtEngine& smte, LogicInfo& logic)
     options::bvLazyRewriteExtf.set(false);
   }
 
-  if (!options::sygusExprMinerCheckUseExport())
```

Comment:
How is this related to this PR?

Reply:
See comment explaining why this option is no longer necessary.